### PR TITLE
dependencies and webpacker

### PR DESCRIPTION
**Pull request by seefood**
Things I bumped into while setting up and them updating my dev env.

### CODE REVIEW

File: Running-Mastodon/Development-guide.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ In fact, all you need is described in the [production guide](Production-guide.md
 - To prefix any commands with `RAILS_ENV=production` since the default environment is "development" anyway
 - Any cronjobs
 
+Ubuntu dependencies for a development environment (bundler will fail if these are not installed in advance)
+
+    sudo apt-get install libpq-dev libssl-dev libprotobuf-dev protobuf-compiler ; sudo gem update --no-ri --no-rdoc
```
Comment (Gargron) [inline Running-Mastodon/Development-guide.md]: I'm not sure `gem update` is required on a new system, in fact I'm pretty sure it isn't.

File: Running-Mastodon/Development-guide.md [hunk 2]
```diff
@@ -27,8 +31,14 @@ To setup the `mastodon_development` database, run:
...
 
+    ./bin/webpack-dev-server & # New after 1.3.3
     bundle exec rails server
 
+Another approach to manage the different servers starting (redis, rails, webpacker, Sidekiq, streaming):
+
+    gem install foreman --no-ri --no-rdoc
+    foreman start
+
 And open `http://localhost:3000` in your browser. Background jobs run inline (aka synchronously) in the development environment, so you don't need to run a Sidekiq process.
```
Comment (Gargron) [inline Running-Mastodon/Development-guide.md]: >Background jobs run inline (aka synchronously) in the development environment, so you don't need to run a Sidekiq process.

No longer true

File: Running-Mastodon/Development-guide.md [hunk 3]
```diff
@@ -27,8 +31,14 @@ To setup the `mastodon_development` database, run:
 
 You can then run Mastodon with:
 
+    ./bin/webpack-dev-server & # New after 1.3.3
```
Comment (Gargron) [inline Running-Mastodon/Development-guide.md]: Probably not ideal to detach this process, log output will be really messy. Just recommend using different tabs in the terminal or two terminals. If someone wants to use detaching, they'll know how to anyway.

File: Running-Mastodon/Development-guide.md (hunk 2 above)
Comment (aral) [inline Running-Mastodon/Development-guide.md]: Is there anything specific we need to do manually to run the Sidekiq process or does foreman handle that? (i.e., should that sentence just be removed?)

**Pull request merged**